tests/utils.go

```diff
@@ -1785,6 +1799,30 @@ func removeNamespaces() {
 	}
 }
 
+func removeAllGroupVersionResourceFromNamespace(groupVersionResource schema.GroupVersionResource, namespace string) error {
+	virtCli, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return err
+	}
+
+	gvr, err := virtCli.DynamicClient().Resource(groupVersionResource).Namespace(namespace).List(context.Background(), metav1.ListOptions{})
+	if errors.IsNotFound(err) {
+		// gvr is not found, no-op
```

Comment:
I think there's hardly any motivation for this comment; it's pretty obvious what you're doing.

No need to update the PR if it's just this; only consider it when another reviewer complains about something.